projects/storefrontlib/src/cms-components/navigation/search-box/search-box.component.spec.ts

```diff
@@ -262,6 +289,54 @@ describe('SearchBoxComponent', () => {
       ).toBeTruthy();
     });
 
+    it('should contain chosen word from the dropdown', () => {
+      const mockRouterState = {
```

Comment:
let's move this object as 'global' and settings the params property as an empty object to be used here https://github.com/SAP/spartacus/pull/8298/files#diff-8713f6e9795265679e836cf3c74e49434601908590027faf3be387cb6e4ee5d0R72

After, you can just re-define an object using that mockedObject and spread it with the value you have in params